app/tools/android/make_apk.py

```diff
@@ -317,9 +317,10 @@ def CopyCompressedLibrary(native_path, library_path, raw_path, arch):
   # need dummy library file to keep the arch info if enable lzma.
```

Comment:
Does the comment need to be updated after this change? It sounds like the dummy library file must be there if LZMA support is enabled, which apparently does not seem to be the case.